foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -95,6 +98,37 @@ public static OpenAiClient withCustomDestination(@Nonnull final Destination dest
     return new OpenAiClient(destination);
   }
 
+  /**
+   * Add a system prompt before user prompts.
+   *
+   * @param systemPrompt the system prompt
+   * @return the client
+   */
+  @Nonnull
+  public OpenAiClient withSystemPrompt(@Nonnull final String systemPrompt) {
+    parameters =
+        new OpenAiChatCompletionParameters()
```

Comment:
We could move this to the constructor. This has the advantage that the user could also configure other parameters in the future, instead of giving them with each prompt

Reply:
The constructor is private